spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/db/migration/mysql/V3__Add_Task_Metadata_Table.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.db.migration.mysql;
+
+import java.util.Arrays;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+
+import org.springframework.cloud.dataflow.server.db.migration.SqlCommand;
+import org.springframework.cloud.dataflow.server.db.migration.SqlCommandsRunner;
+
+/**
+ * Adds the database table to store task manifests in (one for each execution).
+ *
+ * @author Michael Minella
+ * @since 2.3
+ */
+public class V3__Add_Task_Metadata_Table extends BaseJavaMigration {
```

Comment:
For 2.3.0 we've added V2 so we should not add V3. Use existing `V2__Add_Descriptions_And_OriginalDefinition.java` and change class name to have better description. And we then need new schemas for other db's as well.

Reply:
Going forward, can we document that these versions apply to a specific version of SCDF?  I added mine because there is no indication of when the V2 was added so I didn't know if I should piggy back on it or make it independent...

Reply:
Done